### PR TITLE
server-1839 | Review PV resize docs for server 4.x release

### PR DESCRIPTION
**Pull request by atulsingh0**
# Description
A brief description of the changes.

# Reasons
[SERVER-1839](https://circleci.atlassian.net/browse/SERVER-1839)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by rosieyohannan** [comment]
Thank you @atulsingh0! I am about to change all the filenames in #6993 but I will push a change to this branch too to move your changes over to the right place once that's done.

File: jekyll/_cci2/server/Operator/extending-internal-volumes.adoc [hunk 1]
```diff
@@ -93,18 +93,18 @@ gp2 (default)   kubernetes.io/aws-ebs   Delete          WaitForFirstConsumer   t
...
 ----
 
 For MongoDB:
 [source,bash]
 ----
-kubectl delete sts mongodb --cascade=false
+kubectl delete sts mongodb --cascade=orphan
 ----
 
 === Step 3 - Update the size of the database's PVC
```
Comment (lloydChris) [inline jekyll/_cci2/server/Operator/extending-internal-volumes.adoc:110]: Is Step 3 still required?  Do we really need to patch the PVC before doing a helm upgrade with the new volume size?

Comment (atulsingh0) [reply jekyll/_cci2/server/Operator/extending-internal-volumes.adoc:110]: Yes, Step 3 is still required.

**Comment by rosieyohannan** [comment]
@atulsingh0 I've added your changes to a new PR if you could double check it #7009

**Pull request closed without merging**